examples/experimental/dagster-airlift/examples/dbt-example/dbt_example/dagster_defs/lakehouse.py

```diff
@@ -36,7 +37,19 @@ def _multi_asset() -> None:
             columns=columns,
         )
 
-    return Definitions(assets=[_multi_asset])
+    return _multi_asset
+
+
+def defs_from_lakehouse(
```

Comment:
should this still be around? This could only get used with Definitions.merge right?

Reply:
It's used in `complete.py`. Deleted upstack